src/styles/block-card.css

```diff
@@ -0,0 +1,97 @@
+/* #region Card */
+.card {
+  display: flex;
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  flex-direction: column;
+  color: #060b35;
+  --button-color: #00acdc;
```

Comment:
More corretcly declare variables in _:root_ selector

Reply:
@YegorKorenkov But if i will want to transfer this block, these variables must to be inside the block .card